index.js

```diff
@@ -449,7 +449,8 @@ class ScmBase {
                 createTime: Joi.reach(dataSchema.core.scm.pr, 'createTime'),
                 url: Joi.reach(dataSchema.core.scm.pr, 'url'),
                 userProfile: Joi.reach(dataSchema.core.scm.pr, 'userProfile'),
-                baseBranch: Joi.reach(dataSchema.core.scm.pr, 'baseBranch')
+                baseBranch: Joi.reach(dataSchema.core.scm.pr, 'baseBranch'),
+                mergeable: Joi.boolean().required()
```

Comment:
Dose this change effects gitlab and bitbucket?

Reply:
This change has no effect on them.

Because of
- `scm-gitlab` don't override `_getPrInfo`
- `_getPrInfo` of `scm-bitbucket ` return `null`

Reply:
Does `required()` allow `null`?

Reply:
I fixed it.